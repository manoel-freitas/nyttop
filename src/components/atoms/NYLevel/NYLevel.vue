<template functional>
  <nav :class="['level', props.extraClasses]">
    <slot />
  </nav>
</template>
