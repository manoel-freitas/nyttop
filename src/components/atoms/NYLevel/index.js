import NYLevel from './NYLevel'

export default NYLevel
