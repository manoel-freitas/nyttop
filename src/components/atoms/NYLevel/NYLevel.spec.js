/// <reference types="jest" />

import { mount } from '@vue/test-utils'
import NYLevel from './NYLevel'

const nyLevel = mount(NYLevel, {
  context: {}
})
test('NYLevel has to be mounted', () => {
  expect(nyLevel.is(NYLevel)).toBe(true)
})

test('snapshot test', () => {
  expect(nyLevel.element).toMatchSnapshot()
})
