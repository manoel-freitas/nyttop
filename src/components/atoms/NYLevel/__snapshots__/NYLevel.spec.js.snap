// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<nav
  class="level"
/>
`;
